### PR TITLE
Optimize config.xml plugins spec

### PR DESCRIPTION
**Pull request by vitorsemeano**
This should fix issues when running npx cordova prepare.

Also i removed the submodule cordova-plugin-chromecast from the filesystem, in addition to including it directly using config.xml.

branch master was also added to .submodules for the submodule jellyfin-web.

package.json and config.xml are synced when cordova prepare run. So i found plugin spec compatible with npm and cordova, for nativeshell and the chromecast plugins.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,6 +1,4 @@
 [submodule "src/jellyfin-web"]
 	path = src/jellyfin-web
 	url = https://github.com/jellyfin/jellyfin-web.git
-[submodule "src/cordova-plugin-chromecast"]
-	path = src/cordova-plugin-chromecast
-	url = https://github.com/jellyfin/cordova-plugin-chromecast.git
+	branch = master
```
Comment (thornbill) [inline .gitmodules]: Files should end with a newline.

**Comment by vitorsemeano** [comment]
For the build file, i removed all references to submodule chromecast.

To preserve the changes made by Joshua Boniface, i adapted the loading of the submodule jellyfin-web. So if --web-branch or -b exists, then does a checkout since it was already working, and if nothing is present, then proceed in updating the submodule with the most recent commit from master.

File: .gitmodules (hunk 1 above)
Comment (vitorsemeano) [reply .gitmodules]: Hi, it's fixed, please check it, thanks.

**Comment by joshuaboniface** [comment]
Great changes, thanks @vitorsemeano!

**Pull request merged**